### PR TITLE
Feature/infite scroll marketplace

### PR DESCRIPTION
**Pull request by mmiranda-stackit**

### CODE REVIEW

File: src/containers/explore-product/index.tsx [hunk 1]
```diff
@@ -20,12 +21,13 @@ const LoadingSpinner = () => (
...
-    if (nfts.length >= 10) {
+  const loadMoreItems = async () => {
+    const { data } = await axios.get('http://localhost:3334/v1/items?limit=10');
+    if (nfts.length >= data.itemsLength) {
       setHasMore(false);
     }
-
-    console.log('finish');
+    console.log(data.items);
+    console.log(data.itemsLength);
```
Comment (juliswer) [inline src/containers/explore-product/index.tsx]: Remove console.log

File: src/types/item.ts [hunk 2]
```diff
@@ -23,4 +23,5 @@ export type Item = {
   status: number;
   tokenId: number;
   price: string;
+  items: any;
```
Comment (juliswer) [inline src/types/item.ts:28]: Please assign a type to items

File: src/components/item-filter/index.tsx [hunk 3]
```diff
@@ -1,60 +1,27 @@
...
-
-  const [isOpen, setIsOpen] = useState(false as boolean);
+import useAppSelector from '../../store/hooks/useAppSelector';
+import {
+  selectNFTsMarketplace,
+  setMarketplaceFilters,
+} from '../../features/marketplace/store/marketplace.slice';
+
+// TODO: Get cheapest and most expensive prices
+const cheapest = 0.00001;
```
Comment (juliswer) [inline src/components/item-filter/index.tsx]: Hardcode?

File: src/features/leda-nft/store/leda-nft.slice.ts [hunk 4]
```diff
@@ -61,65 +61,10 @@ const ledaNftSlice = createSlice({
   },
 });
 
-// TODO: Change this name to selectNftState
 export const selectNftState = (state: RootState) => state.ledaNft;
 
 export const selectAllItems = (state: RootState) => state.ledaNft.items;
 
-export const selectFilteredItems = createSelector(
```
Comment (juliswer) [inline src/features/leda-nft/store/leda-nft.slice.ts:85]: looks good :(

**Review by juliswer — COMMENTED** [review]
I let you many comments. If they are already solved, do merge to main. Otherwise, please correct them and let me know so I approve the PR.
Great job man!

File: src/containers/explore-product/index.tsx (hunk 1 above)
Comment (mmiranda-stackit) [reply src/containers/explore-product/index.tsx]: Where did you take this code? This component was removed

File: src/components/item-filter/index.tsx (hunk 3 above)
Comment (mmiranda-stackit) [reply src/components/item-filter/index.tsx]: Are you sure you are reviewing the latest commit?

**Review by juliswer — APPROVED** [review]
Looks good

File: src/features/marketplace/store/marketplace.slice.spec.ts [hunk 5]
```diff
@@ -11,6 +18,19 @@ describe('Marketplace slice', () => {
     initialState = {
       owner: '',
       isLoading: false,
+      marketplaceFilters: {
```
Comment (AlefrankM) [inline src/features/marketplace/store/marketplace.slice.spec.ts:21]: I don't like this hardcode

**Pull request merged**